sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -1180,9 +1180,9 @@ proces %2, a połączył się proces %3.%n%n
 Jeśli ta usługa jest skonfigurowana do uruchomienia z debugerem, takie zachowanie jest oczekiwane.
 .
 Language=Romanian
-Procesul serviciului, altul decât ceea ce este pornit în Administratorul de control al serviciilor
-e conectat odată cu pornirea serviciului %1. Administratorul de control al serviciilor a lansat, în schimb
-procesul %2 și procesul %3 conectat.%n%n
+Un proces al serviciului, altul decât ceea ce este pornit în Administratorul de control al serviciilor
```

Comment:
Here, the English "service process" is describing the type of the process, not that the belongs to a service. So here's a suggestion: "Un proces de tip serviciu" instead of "Un proces al serviciului".